third_party/terraform/utils/provider_test.go.erb

```diff
@@ -83,7 +83,9 @@ func init() {
 	// if the example is versioned as beta; normal beta tests should continue to
 	// use the standard provider map.
 	testAccProvidersOiCS = map[string]terraform.ResourceProvider{
-		"google-beta": testAccProvider,
+<% Api::Product::Version::ORDER[1..Api::Product::Version::ORDER.index(version)].each do |lower_version| -%>
```

Comment:
Do you mind explaining what we're doing with a comment here? It's a little hard to parse amidst the `Api::Product::Version::ORDER`s in there.